src/microbe_stage/Microbe.Interior.cs

```diff
@@ -106,6 +113,9 @@ public partial class Microbe
     [JsonProperty]
     public int AgentVacuoleCount { get; private set; }
 
+    [JsonProperty]
+    public List<SlimeJetComponent> SlimeJets { get; private set; } = new();
```

Comment:
Then the SlimeJet components don't have to be stored in the Microbe class. IMHO it would probably quickly turn into a mess otherwise if we have to do this for many other possible components.

Reply:
I haven't looked at the overall design but I'll cautiously say I'm in favour of this. Perhaps a design like the flagellum could be used if multiple slime jets help with overall propulsion to avoid having each slime jet call separately into the clouds system.